airgun/views/common.py

```diff
@@ -56,6 +62,33 @@ class listremove(SatSecondaryTab):
         './/nav[@class="ng-scope"]/following-sibling::div')
 
 
+class LCESelectorGroup(ParametrizedView):
+    ROOT = ".//div[@path-selector='environments']"
+
+    PARAMETERS = ('lce_name',)
+
+    LAST_ENV = ".//div[contains(@class, 'path-selector')]/ul/li[last()]"
+    lce = LCESelector(
+        locator=ParametrizedLocator(
+            "./div[contains(@class, 'path-selector')]/ul"
+            "[li[normalize-space(.)='{lce_name}']]")
+    )
+
+    @classmethod
+    def all(cls, browser):
+        return [(element.text,) for element in browser.elements(cls.LAST_ENV)]
```

Comment:
ideally `element.text` should be replaced with `browser.text(element)`, but it's currently blocked by RedHatQE/widgetastic.core#92 , not sure whether we should block this PR or introduce improvement later.